code-challenges.test.js

```diff
@@ -13,55 +13,180 @@
+    fibNumbersArray[index] = fibNumbersArray[index - 1] + fibNumbersArray[index - 2]
+  }
+  return fibNumbersArray
+}
+
+// PASS  ./code-challenges.test.js
+// fibonacci
+//   ✓ takes in a number greater than 2, and returns an array that length containing the numbers of the Fibonacci sequence (2 ms)
+
+
+
+
+
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("objectsArraySorted", () => {
+  it("takes in an object and returns an array of the values sorted from least to greatest", () => {
+    const studyMinutesWeek1 = {
+      sunday: 90,
+      monday: 30,
+      tuesday: 20,
+      wednesday: 15,
+      thursday: 30,
+      friday: 15,
+      saturday: 60
+    }
+    // Expected output: [15, 15, 20, 30, 30, 60, 90]
+    const studyMinutesWeek2 = {
+      sunday: 100,
+      monday: 10,
+      tuesday: 45,
+      wednesday: 60,
+      thursday: 20,
+      friday: 15,
+      saturday: 65
+    }
+    // Expected output: [10, 15, 20, 45, 60, 65, 100]
+    expect(objectsArraySorted(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+    expect(objectsArraySorted(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+  })
+})
+
+// FAIL  ./code-challenges.test.js
+// fibonacci
+//   ✓ takes in a number greater than 2, and returns an array that length containing the numbers of the Fibonacci sequence (3 ms)
+// objectsArraySorted
+//   ✕ takes in an object and returns an array of the values sorted from least to greatest
+
+// ● objectsArraySorted › takes in an object and returns an array of the values sorted from least to greatest
+
+//   ReferenceError: objectsArraySorted is not defined
 
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
+/* PSEUDOCODE
+    -create a function named objectsArraySorted
+    - input: it will take in an object
+    - process: 
+        -take in the object, and convert it into an array using object.values method
+        -once that object is an array, use the .sort method to sort it from smallest to largest
+        -use a comparison function to sort the indexes properly.
+    - output: an array with number values sorted from least to greatest.
+*/
+
+const objectsArraySorted = (object) => {
+  let objectArray = Object.values(object)
+  return objectArray.sort((value, index) => value - index)
```

Comment:
I love the fact that you found how to use a comparison function to sort from least to greatest. 👏🏽 I would consider changing the names of the parameters you are using because it might lead someone to think you are referencing the value and index to determine how it is sorted and it works a bit differently.